docs/netdata-agent/versions-and-platforms.md

```diff
@@ -1,116 +1,229 @@
 |:---------------:|:---------------------------------------------:|:---------------------------------------------------------:|:----------------------------------------:|:---------------------------------------------------------------------------------:|
-|     Stable      | At most once per month, usually every 45 days | Receiving bug fixes and security updates between releases | Up to the 2nd stable release after them  |     Previous configuration semantics and data are supported by newer releases     |
-|     Nightly     |           Every night at 00:00 UTC            |               Latest pre-released features                | Up to the 2nd nightly release after them | Configuration and data of unreleased features may change between nightly releases |
+|   **Stable**    | Usually 4-6 major/minor releases per year plus patch releases as needed | Receiving bug fixes and security updates between releases | Up to the 2nd stable release after them  |     Previous configuration semantics and data are supported by newer releases     |
+|   **Nightly**   |         Most nights around 02:00 UTC          |               Latest pre-released features                | Up to the 2nd nightly release after them |Configuration and data of unreleased features may change between nightly releases|
 
-> "Support Duration" defines the time we consider the release as actively used by users in production systems, so that all features of Netdata should be working like the day they were released. However, after the latest release, previous releases stop receiving bug fixes and security updates. All users are advised to update to the latest release to get the latest bug fixes.
+:::info  
+
+**Support Duration** defines how long we consider each release actively used in production systems. After this period, you should update to the latest release to continue receiving bug fixes and security updates.
+
+:::
+
+## Switching Between Stable and Nightly Builds
+
+You can switch between stable and nightly channels depending on your needs. The method depends on how you originally installed Netdata.
+
+<details>
+<summary><strong>For Native Package Installations</strong></summary><br/>
+
+If you installed Netdata using our native packages (RPM, DEB), you can switch channels by updating your repository configuration:
+
+**Method 1: Using Package Manager (Recommended)**
+
+```bash
+# Switch from nightly to stable
+sudo apt install netdata-repo  # This will uninstall netdata-repo-edge automatically
+# Or for RPM-based systems:
+sudo yum install netdata-repo  # This will uninstall netdata-repo-edge automatically
+```
+
+**Method 2: Manual Repository Update**
+
+1. Update your repository configuration to point to the desired channel
+2. Reinstall the Netdata package using your system package manager
+
```

Comment:
This approach should not be recommended, it is very likely to break the next time we publish a new version of the repository configuration packages.